x/profile/keeper/msg_server_update_coordinator_address_test.go

```diff
@@ -62,6 +72,13 @@ func TestMsgUpdateCoordinatorAddress(t *testing.T) {
 				NewAddress: coord1.Address,
 			},
 			err: types.ErrCoordAlreadyExist,
+		}, {
+			name: "inactive coordinator",
+			msg: types.MsgUpdateCoordinatorAddress{
+				Address:    disableCoord.Address,
+				NewAddress: addr2,
+			},
+			err: types.ErrCoordAddressNotFound,
```

Comment:
It should be `types.ErrCoordInactive`, the fact that the test is passing with a different error means the code is invalid
In this case, it is because `GetCoordinatorByAddress` the inactive validator
So it makes more sense I think to use `GetActiveCoordinatorByAddress `

Reply:
So, as things are, only the `Coordinator` message has the `Active` field.  If we run `GetCoordinatorByAddress`, and we don't find the `CoordinatorByAddress`, then there is no way to then find the `Coordinator` and check if it is active.  

So if this is the case, the only error we can return is `ErrCoordAddressNotFound`.  The Coordinator could have never been created OR could have been disabled and removed.  

In other words, we can't definitively say if the Coordinator is Active if we are trying to access it by the Address (instead of ID).  

If this kind of behavior is not desired, and we want to always be able to say if the Coordinator is Active, Inactive, or Not Created, we will have to no longer remove the `CoordinatorByAddress` entry when disabling.  We can definitely do this, but I want to see your thoughts.  Initially @lubtd suggested we continue removing the `CoordinatorByAddress` entry on removing. 

Thoughts?

Reply:
Yes, we must keep removing `CoordinatorByAddress` on disable because it enable the user to create a new coordinator, this is the right behavior.

As mentioned in the general comment in my review, we can remove test case that assert solely on inactive coordinator for methods using `GetCoordinatorByAddress`/`GetActiveCoordinatorByAddress` because `GetCoordinatorByAddress`/`GetActiveCoordinatorByAddress` is already tested for not retrieving inactive coordinator so a single test case when the coordinator is not found is sufficient